File: OLD_VERSION/Beta_1_FrontEnd_Angular/src/app/body/api/consultation/StereoscopicVision.ts
export interface StereoscopicVision {
    id: number;
    createdBy: string;
    createdAt: string;
    modifiedBy: string;
    modifiedAt: string;
    deleted: boolean;
    version: number;

    synoptophore: number | null;
    randot: number | null;
    nearOther: number | null;
    titmus: number | null;
    tno: number | null;
    distanceOther: number | null;
}

File: final_back_end_spring_boot/src/main/java/com/jorami/eyeapp/model/consultation/StereoscopicVision.java
package com.jorami.eyeapp.model.consultation;

import com.jorami.eyeapp.model.IdentifiedModel;
import jakarta.persistence.Entity;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.hibernate.annotations.Where;
import org.hibernate.envers.Audited;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Audited
@Where(clause = "deleted = FALSE")
@Entity
@Table(name="stereoscopic_vision")
public class StereoscopicVision extends IdentifiedModel {
    private Float synoptophore;
    private Float randot;
    private Float nearOther;
    private Float titmus;
    private Float tno;
    private Float distanceOther;
}
